tests/minifier.js

```diff
@@ -658,7 +672,7 @@
     equal(minify(input, { collapseWhitespace: true }), output);
 
     input = '<p> foo    <span>  blah     <i>   22</i>    </span> bar <img src=""></p>';
-    output = '<p>foo <span>blah <i>22</i></span> bar <img src=""></p>';
+    output = '<p>foo <span>blah <i>22</i> </span>bar <img src=""></p>';
```

Comment:
It's removing the space after `</span>` rather than the one before.

If there isn't any space before `</span>` then this wouldn't have happened.